docs/software/media.md

```diff
@@ -90,7 +90,8 @@ Also installs:
     Make sure you have one of DietPi's [File Servers](../file_servers/) installed.  
     Default music directory:
 
-    - `/mnt/dietpi_userdata/Music`, `/Music` from NFS/Samba
+    - `/mnt/dietpi_userdata/Music`, `/Music` from NFS/Samba  
+       (i.e. if you want to add a remote share this need to be mounted below `/mnt/dietpi_Userdata/Music`)
```

Comment:
Actually this is meant for File "servers" not remote shares/mounts. `/mnt/dietpi_userdata/Music`, `/Music` is the default for NFS and Samba servers on DietPi, so this can be used when moving files from remove NFS/Samba clients onto the DietPi media server.

When mounting remote network shares on DietPi, of course any other directory can be used. Some media software have `/mnt/dietpi_userdata/Music` used as default music/audio/playlist dir, but not all of them, and usually this can be changed/new locations added trivially. So "this need to be mounted" gives a false impression, I'd say, and I'd generally not mount shares below this directory, but keep it for local media only, but either way works, of course.

Reply:
Changed back with the latest push.
Also added some more configuration information at all MPD based frontends.